src/module_PI/Raspberry_PI/main/EmailNotifier.java

```diff
@@ -37,11 +38,13 @@ public EmailNotifier(String recipientEmailAddress, String message, String pathTo
 	}
 
 	private void loadGmailCredentials() {
-			gmailUsername = PiApp.GMAIL_CREDENTIALS.getProperty("gmailUsername");
-			gmailPassword = PiApp.GMAIL_CREDENTIALS.getProperty("gmailPassword");
+		PiApp.LogToFile("EmailNotifier credentials have been loaded");
```

Comment:
Shouldn't this be logged after the properties are actually set? Its conceivable that something goes wrong in loading these properties so you don't want to 'tell the future' with your log statements. You could change the way it is worded and have it say `EmailNotifier credentials are being loaded` so you are saying what is coming next instead of assuming that it will work properly.